gradle/libs.versions.toml

```diff
@@ -1,32 +1,32 @@
 [versions]
-asset-pipeline = '4.3.0'
+asset-pipeline = '5.0.0'
 commons-lang = '2.6'
-ehcache = '2.10.9.2'
-geb = '6.0'
-gorm-hibernate5 = '8.0.3'
-grails = '6.1.2'
-grails-async-and-events = '5.0.2'
-grails-datamapping = '8.0.3'
-grails-gsp = '6.2.0'
-grails-converters = '4.0.1'
-grails-testing-support = '3.1.2'
-groovy = '3.0.21'
-h2database = '2.2.224'
-javax-annotation-api = '1.3.2'
-javax-servlet-api = '4.0.1'
-micronaut = '3.10.4'
-selenium = '4.18.1'
-selenium-safari = '4.18.1'
-slf4j = '1.7.36'
-spock = '2.0-groovy-3.0'
-spring = '5.3.33'
-springboot = '2.7.18'
-spring-security = '5.8.11'
-tomcat = '9.0.87'
+ehcache = '3.10.8'
+geb = '7.0'
+gorm-hibernate5 = '9.0.0-SNAPSHOT'
+grails = '7.0.0-SNAPSHOT'
+grails-async-and-events = '7.0.0-SNAPSHOT'
+grails-datamapping = '9.0.0-SNAPSHOT'
+grails-gsp = '7.0.0-SNAPSHOT'
+grails-converters = '6.0.0-SNAPSHOT'
+grails-testing-support = '4.0.0-SNAPSHOT'
+groovy = '4.0.22'
+h2database = '2.3.232'
+jakarta-annotation-api = '3.0.0'
+jakarta-servlet-api = '6.0.0'
+micronaut = '4.5.3'
+selenium = '4.19.1'
+selenium-safari = '4.19.1'
+slf4j = '2.0.16'
+spock = '2.4-M4-groovy-4.0'
```

Comment:
Probably better to use `2.3-groovy-4.0` and not a milestone release. Or is there a reason why you used this version?

Reply:
Done.